libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -574,8 +574,16 @@ export abstract class BaseInstallManager {
         args.push('--network-concurrency', options.networkConcurrency.toString());
       }
 
-      if (this._rushConfiguration.pnpmOptions.strictPeerDependencies) {
-        args.push('--strict-peer-dependencies');
+      if (semver.gte(this._rushConfiguration.pnpmVersion, '7.0.0')) {
```

Comment:
use `this._rushConfiguration.packageManagerToolVersion` since at this point you already know that the tool is PNPM.

Reply:
I see. Ok. I was not aware of this. Will do so